usaspending_api/awards/v2/filters/matview_filters.py

```diff
@@ -293,22 +293,19 @@ def keyword_parse(keyword):
 
         # Federal Account Filter
```

Comment:
With removing the `faba_flag` should this comment be removed? The same thing below.

Reply:
No, because these are still the filters used by the federal account page. I assume the comment was denoting that, and not the the flag is about to be set.

Reply:
What is performance like with all of the joins originating from the matviews?

The original SQL we discussed yesterday used the `faba_flag` for a separate query (unsure if it was run beforehand or as a sub-query).

Reply:
Quite good, based on the testing against DEV DB shown in the PR description. The framework that was already partially set up was VERY slow (~30 seconds in local DB), likely due to the giant IN clause. This was rearranged to mimic what v1 was doing, and seems to still be performant for the use cases UI is using it for.

Reply:
Did you test SpendingByAwardCount? That is the current problem-child

Reply:
Yes, although only for accuracy, not performance. The latter relied on my linking to DEV DB, which isn't possible now that we have to put the matviews back.

Reply:
hmmm, and staging is swamped now

Reply:
**very** quick checks appear OK, we'll see how they are in staging next week.